CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# dbt_netsuite v0.UPDATE.UPDATE
+# dbt_netsuite v0.8.1
```

Comment:
Even though you are correct that this is technically a non breaking change, I would actually suggest we make this a breaking change since we are removing an intermediate model. Although we are materializing this as ephemeral by default, I am unsure if customers are referencing this model on it's own. I would not want to push a patch update which could potentially be catastrophic for customers who are referencing this model directly on their own.

Because of this I would up the version index to 0.9.0

Reply:
Your reasoning makes sense, I didn't consider that customers could still be referencing the deleted intermediate model on its own! I'll make it a breaking change